model/lti.go

```diff
@@ -94,3 +95,14 @@ func transformLTIUsername(ltiUsername string) string {
 
 	return mattermostUsername
 }
+
+func GetLMSChannelSlug(personalChannelName, channelId string) string {
```

Comment:
Can this function be used elsewhere as well?

Reply:
Done